Clean right panel

Algorush opened the pull request.

Algorush committed:
change design

Algorush committed:
unhide components

unhide components on the right panel

kfarr commented:
This is a great start, thanks for making a clean branch to start these changes.

I would suggest to continue working on updating the styling as part of that same branch, specifically to update the component title and widget fields to match the styling from the figma (highlighted in red)
![image](https://user-images.githubusercontent.com/470477/186340442-0a989705-bf3a-4086-98e9-f33f708c2909.png)

Also the font has reverted to san serif (arial / helvetica style) instead of the desired font, it seems to be part of the changes in the "change design" commit

Algorush committed:
Add styles for components

kfarr commented:
Great progress! Nice to see the improvements in styling of the right panel fields.

One issue: the font still appears in sans serif and not lato. There is an error in the console:
`GET https://fonts.googleapis.com/css2?family=Lato:wght@400;500,600&display=swap net::ERR_ABORTED 400`

Here is a screenshot of what it looks like when I run this PR / branch locally:
![image](https://user-images.githubusercontent.com/470477/188054645-36597bfc-d63f-4753-b3e3-d54e7ec13940.png)

kfarr committed:
fix google font url typo

kfarr committed:
dist build

kfarr committed:
rebase (Merge pull request #23 from 3DStreet/master)

viewer mode usability improvements

Algorush committed:
new styles for hovered Layers and right panel tabs

Algorush committed:
fix some style problems for components

kfarr committed:
Merge branch 'master' into clean-right-panel

kfarr commented:
This is great progress, Alex!

Here are 3 things to fix that I found. I will ask Daria to comment next:

1. The hover state for the layer panel collapse button with left arrow icon does not work (it should show dark purple as the right-hand panel collapse button does correctly)
![image](https://user-images.githubusercontent.com/470477/189467390-ab61fe71-be35-465a-84a3-b8f3060f8a1c.png)

2. The hover state for the layer panel expand button has icon positioning that is offset from the icon position in the non-hover (regular) state. You can see the layers icon "jump" when I rapidly hover and unhover the button. Instead, the icon should be positioned identically for both hover and non-hover states:
https://user-images.githubusercontent.com/470477/189467440-2d92d07a-63dc-4f3b-a680-8f949af9b87a.mov

3. The hover state for the properties panel expand button has icon positioning that is offset from the icon position in the non-hover (regular) state. You can see the properties icon "jump" when I rapidly hover and unhover the button. Instead, the icon should be positioned identically for both hover and non-hover states:
https://user-images.githubusercontent.com/470477/189467523-d452cd28-a426-40cc-9c59-01305463ab50.mov

kfarr commented:
Feedback from Daria ([screenshot from figma](https://www.figma.com/file/W9jid3A0jgssnIKBFEvG38/3DStreet-2022?node-id=1917%3A19038))

![image](https://user-images.githubusercontent.com/470477/189553805-199ab150-d581-438b-9f54-b2d48fdfb7a4.png)

Algorush committed:
fix styles for Layers and Properties panels

fix styles for hover and active states for Layers and Properties tabs
fix style for input

Algorush committed:
fix Layers icon in hover state
